site/docs/spark-queries.md

```diff
@@ -166,16 +168,15 @@ To show the valid snapshots for a table, run:
 ```sql
 SELECT * FROM prod.db.table.snapshots
 ```
-```text
-+-------------------------+----------------+-----------+-----------+----------------------------------------------------+-------------------------------------------------------+
-| committed_at            | snapshot_id    | parent_id | operation | manifest_list                                      | summary                                               |
-+-------------------------+----------------+-----------+-----------+----------------------------------------------------+-------------------------------------------------------+
+
+<div class="markdown-table-container" markdown="block">
+| committed_at | snapshot_id | parent_id | operation | manifest_list | summary |
+| -- | -- | -- | -- | -- | -- |
 | 2019-02-08 03:29:51.215 | 57897183625154 | null      | append    | s3://.../table/metadata/snap-57897183625154-1.avro | { added-records -> 2478404, total-records -> 2478404, |
 |                         |                |           |           |                                                    |   added-data-files -> 438, total-data-files -> 438,   |
```

Comment:
This has 3 rows but seems to only contain data for a single snapshot. Here's what it looks like when I `mkdoc serve` locally:
<img width="773" alt="Screen Shot 2021-12-28 at 4 43 05 PM" src="https://user-images.githubusercontent.com/43911210/147609280-03d9197e-78f2-417c-a905-5ff9de440bd3.png">

Reply:
Is this perhaps what you meant?
```
<div class="markdown-table-container" markdown="block">
| committed_at | snapshot_id | parent_id | operation | manifest_list | summary |
| -- | -- | -- | -- | -- | -- |
| 2019-02-08 03:29:51.215 | 57897183625154 | null      | append    | s3://.../table/metadata/snap-57897183625154-1.avro | { added-records -> 2478404, total-records -> 2478404, added-data-files -> 438, total-data-files -> 438, spark.app.id -> application_1520379288616_155055 } |
| ...                     | ...            | ...       | ...       | ...                                                | ...                                                   |
</div>
```

Reply:
yes, I fix it